Dockerfile

```diff
@@ -7,21 +9,22 @@ ARG OSNICK=buster
 ARG ARCH=x64
 
 #----------------------------------------------------------------------------------------------
-FROM redisfab/redis:6.0-rc3-${ARCH}-${OSNICK} AS builder
+FROM redisfab/redis:${REDIS_VER}-${ARCH}-${OSNICK} AS builder
+
+ARG REDIS_VER
```

Comment:
Why is this here? and in line 27?

Reply:
It's the way Docker handles arguments. You need to re-declare them after each FROM clause. Really annoying.